ibis/pandas/execution/generic.py

```diff
@@ -875,8 +875,17 @@ def execute_node_where_scalar_scalar_series(op, cond, true, false, **kwargs):
 @execute_node.register(
     ibis.pandas.client.PandasTable, ibis.pandas.client.PandasClient
 )
-def execute_database_table_client(op, client, **kwargs):
-    return client.dictionary[op.name]
+def execute_database_table_client(op, client, timecontext=None, **kwargs):
```

Comment:
Can you remove the default value for timecontext here?

Reply:
removed